app/scripts/lib/assertion.js

```diff
@@ -24,14 +24,13 @@ function (P, jwcrypto) {
     // browser has native crypto, no need to fetch entropy from the server.
     try {
       if (window.crypto && window.crypto.getRandomValues) {
-        return P(true); /*jshint ignore:line*/
+        return P(true);
       }
     } catch(e) {
       // some browsers blow up when trying to query window.crypto.
     }
 
     // wah wah, we need to get entropy from the server.
-    //jshint validthis: true
```

Comment:
@IanVS @TDA Great tip, thanks for the heads up. I still haven't looked at the [breaking] changes in **eslint@1** (versus our current v0.24.1), but yeah, I'll try and normalize all these in the eslint-config-fxa module, plus make sure we still have our solid set of default rules since I think the new ESLint 1.x will reset rules by default. Then once ESLint v1 is final (and the related grunt-eslint is updated), we can update our ~10 repos.

Ah, looks like I need to read this today: http://eslint.org/docs/1.0.0/user-guide/migrating-to-1.0.0